test/controllers/v1/permission_grant_controller_test.exs

```diff
@@ -2,6 +2,12 @@ defmodule Cog.V1.PermissionGrantController.Test do
   use Cog.ModelCase
   use Cog.ConnCase
 
+  # When these tests were written, permissions were allowed to be
+  # associated with users, groups, or roles. This model has been
+  # simplified to only allow permissions to be associated with roles,
+  # but the parameterized test scaffolding has been retained for
+  # the time being. More information about this structure is below.
+  #
```

Comment:
I left it there because we intentionally aren't changing the underlying data model. Also, I wanted to keep the tests as close to the original as possible to have the maximum amount of confidence that may changes to controllers and routes were correct and not just passing because I had changed the behavior of the tests. I'm open to a backlog item to refactor these, though.